Makefile

```
@@ -269,6 +290,15 @@ benchmark-matmult: examples/benchmark/benchmark-matmult.cpp build-info.h ggml.o
 vdot: pocs/vdot/vdot.cpp ggml.o $(OBJS)
 	$(CXX) $(CXXFLAGS) $^ -o $@ $(LDFLAGS)
 
+mulmat-tune.o: examples/mulmat-tune/mulmat-tune.c
+	$(CC)  $(CFLAGS) -c $< -o $@
+
+mulmat-tune: examples/mulmat-tune/mulmat-tune-tool.c ggml.o $(OBJS)
```

```suggestion
mulmat-tune-tool: examples/mulmat-tune/mulmat-tune-tool.c ggml.o $(OBJS)
```

Just use the name of the source file for the program, you're asking for trouble otherwise.

---

renamed mulmat-tune-tool.c as mulmat-tune.c